source/application/SXRMB/SXRMB.h

```diff
@@ -19,7 +19,9 @@
 namespace SXRMB {
 
 	enum SXRMBErrorCodes {
-		ErrorSXRMBBeamlineShuttersTimeout = 290101  // 2-CLS 9-SXRMB 01 module-01 errorID-01
+		ErrorSXRMBBeamlineShuttersTimeout = 290101,  // 2-CLS 9-SXRMB 01 module-01 errorID-01
```

Comment:
@dretrex @davidChevrier yes, we don't have a solution yet. This is what I have already before our discussion. I simply follow what I have had already. It will be an easy fix when we have a solution.